buildpack.md

```diff
@@ -301,17 +312,19 @@ At the end of each individual buildpack's build phase:
 
 ### Purpose
 
-The purpose of detection is to find an ordered group of buildpacks to use during the build phase.
-These buildpacks must be compatible with the app.
+The purpose of detection is to find an ordered group of image extensions and buildpacks to use during the build-ext and build phases.
+These image extensions and buildpacks must be compatible with the app.
```

Comment:
```suggestion
```

I think we should remove this phrase, given that the `must` suggests a requirement, and given that "compatible" isn't well-defined.